Create CODE_OF_CONDUCT.md (#399)
stac-utils/pystac: CODE_OF_CONDUCT.md

> # pystac Community Code of Conduct

Reviewer: Spelling elsewhere is PySTAC when used in this sort of context

```suggestion
# PySTAC Community Code of Conduct
```

## Our Pledge
In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be

> reported by contacting the project team at [stac-psc-admins](mailto:stac-psc@radiant.earth). All

Reviewer: ```suggestion
reported by contacting the [STAC Project Steering Committee](mailto:stac-psc@radiant.earth). All
```

complaints will be reviewed and investigated and will result in a response that
is deemed necessary and appropriate to the circumstances. The project team is
obligated to maintain confidentiality with regard to the reporter of an incident.
Further details of specific enforcement policies may be posted separately.

Project maintainers who do not follow or enforce the Code of Conduct in good
faith may face temporary or permanent repercussions as determined by other
members of the project's leadership.

> pystac follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).

Reviewer: ```suggestion
PySTAC follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
```

> # Stac-Utils Community Code of Conduct
> 
> Stac-Utils follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).

Reviewer 2: ```suggestion
# pystac Community Code of Conduct

pystac follows the [STAC spec Code of Conduct](https://github.com/radiantearth/stac-spec/blob/master/CODE_OF_CONDUCT.md).
```
